weblab/repocache/migrations/0001_initial.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-20 12:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('entities', '0009_remove_entity_visibility'),
```

Comment:
This looks like it would just lose all existing visibility info, rather than migrating the entity visibilities to be version visibilities? Which is probably fine at this early stage of the project, since we don't have much real visibility data, but may also have contributed to my errors.

Reply:
Yeah this is a tricky one as it involves moving data out of the database into somewhere external - it could be done, although I'm not sure it could be easily done in a migration - a script could be written if necessary though.